pkg/render/node.go

```diff
@@ -1254,7 +1300,11 @@ func (c *nodeComponent) nodeEnvVars() []corev1.EnvVar {
 		}
 	} else {
 		// BGP is enabled.
-		nodeEnv = append(nodeEnv, corev1.EnvVar{Name: "CALICO_NETWORKING_BACKEND", Value: "bird"})
+		if c.vppDataplaneEnabled() {
+			nodeEnv = append(nodeEnv, corev1.EnvVar{Name: "CALICO_NETWORKING_BACKEND", Value: "none"})
```

Comment:
Probably worth a comment here saying that BGP is enabled, but that the VPP data plane has its own BGP implementation and so we disabled BIRD anyway. Otherwise this code might be not-obvious for future readers.